fe/fe-core/src/main/java/com/starrocks/privilege/ranger/hive/RangerHiveAccessControl.java

```diff
@@ -153,12 +149,12 @@ private boolean hasPermission(RangerHiveResource resource, UserIdentity user, Pr
         }
 
         RangerStarRocksAccessRequest request = RangerStarRocksAccessRequest.createAccessRequest(resource, user, accessType);
-
         RangerAccessResult result = rangerPlugin.isAccessAllowed(request);
+
         if (result != null && result.getIsAllowed()) {
-            return true;
+            return;
         } else {
-            return false;
+            throw new AccessDeniedException();
```

Comment:
i think we need to specify the type of the returned `AccessDeniedException`, i.e. from which access controller?native or ranger? so that we can format different error msg for different access controller